InfluxPublisher/tests/test_publisher_e2e.sh

```diff
@@ -0,0 +1,125 @@
+
+# load the influx db variables
+source ../../influxdb/influx.env
+# 
+
+# Define color codes
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # default color
+
+#for storing output data from the influx query so we can parse. Deleted at end of run.
+output_file="influx_query_test_res.csv"
+
+sleep 6 # when triggered on an action, ensure it's been up long enought to have generated data
```

Comment:
It does seem like we might be depending on a) installing `jq` to take a sufficiently long time, and b) for `docker-compose up` to take a sufficiently short time, so that the data source is up and running before we get to the end of this sleep. I would be interested if there were a better way -- for example, if there were an easy way to repeatedly check until either it's clear that something is wrong, it's clear that things are OK, or until an extremely pessimistic timeout is reached. But I can't think of a better way myself, so if we keep this as-is then I'm OK with that.